### PR TITLE
add cloud backups desc

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-5146

- Created include file for Cloud cluster backup description - added to Backup and Restore Overview and Run Bulk Ops page
- Added description for Dedicated backups on Restore Data from a Backup page

Staging:
- [Backup and Restore Overview](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#cockroachdb-cloud-backup-types)
- [Run Bulk Operations](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/cockroachcloud/run-bulk-operations.html)
- [Restore Data from a Backup](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/cockroachcloud/backups-page.html?filters=dedicated)

### CODE REVIEW

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:5]: I like the start to these! They're going to be very useful. I think there could be some adjustment to the text to make it clear that Cockroach Labs is running these backups and for what types of cluster. 

For managed-service backups:

> - Managed-service backups: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. 

I think we should also add either a link to the Restore Data from Backups page so customers can find the type and frequency of backup for both serverless and dedicated clusters. Something like: 

> "See the [Restore Data from Backups] page for the type and frequency of backups for {{ site.data.products.serverless }} clusters and {{ site.data.products.dedicated }} clusters."

OR

Add them here as sub-bullets. You'll find in my quick-fix PR #15537  some bullet points to these. It might be worth borrowing those for this include, or creating an include file for them as they will need to be updated regularly. Going with this option will mean waiting for my PR to merge and rebasing on master.

I'll leave you to see what makes sense on that front. I could see advantages to both ways. It might be good to see what these look like in a preview.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:6]: Nice job on the if/else logic, well-placed. 

I discussed the naming of the backup types with Liv again, and she felt we should rename "self-service" to "customer-owned backups". If you could make that change here that would be great. We will share this with Liv after these docs changes to see if she has any feedback too.

On the content here, I might slightly adapt the description to match the changes to the other type, something like the following:

> - Customer-owned backups: You can take manual backups and store them in your cloud storage buckets using `BACKUP`.

With these changes, the party taking the backups is the subject of the sentence for both bullet points and so provides that further distinction. Great to link to the `BACKUP` page 👍 You could also link to the Use Cloud Storage for Bulk Operations on "cloud storage bucket".

File: cockroachcloud/run-bulk-operations.md
Comment (kathancox) [reply cockroachcloud/run-bulk-operations.md:9]: And since the Run Bulk Operations page is showing users how to run their own backups, can you add a sentence to that effect directly on the page after the include reference? 

> The examples on this page demonstrate how to take customer-owned backups.

File: v22.1/backup-and-restore-overview.md
Comment (kathancox) [inline v22.1/backup-and-restore-overview.md:52]: Can you adjust the title here to match the product name https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#product-names, so CockroachDB Cloud?

Also, this should be added to v22.2 of this overview page as well!

**Review by kathancox — COMMENTED** [review]
Hey Ashley, great start! Thanks for taking this issue on, it's going to be really helpful for users.

I've included some feedback below for adjustments to some of the language and guidance around the versions. Let me know if you have any questions!

**Comment by ashleykwtam** [comment]
- Added note on the Backups page for Restore Data from Backups page so customers can find the type and frequency of backup for both serverless and dedicated clusters
- Created include files to partition out the bullet points on the Backups page for easy updates in the future 

Staging:
- [Backup and Restore Overview](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#cockroachdb-cloud-backup-types)
- [Run Bulk Ops](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/cockroachcloud/run-bulk-operations.html)

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:3]: Add a period to the end of each bullet point, since they are complete sentences.

File: _includes/cockroachcloud/ccloud/backup-details-dedicated.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-details-dedicated.md:1]: I wasn't super clear here, I actually intended for the frequency of the backups to be in include files for each of the products. I don't think it is necessary at this point though. 

The content here does not need to be include files. So both:

_includes/cockroachcloud/ccloud/backup-details-dedicated.md
_includes/cockroachcloud/ccloud/backup-details-serverless.md

You can add the content back to the page as it was previously and delete these include files.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:3]: Yes! Like this change! Couple small tweaks:

- Bold the names of each backup type
- Adjust the final sentence to remove the repeat of dedicated/serverless, so: 

> See the Restore Data from a Backup page for the type and frequency of backups for both {{ site.data.products.db }} clusters.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:4]: On this second bullet point, it looks as though you may have forgotten to update the second instance of "self-service backup" to "customer-owned backups". When I move to the overview page, it still shows "self-service backups".

**Review by kathancox — CHANGES_REQUESTED** [review]
Hey Ashley, just a couple of small details here to change. Once you push your next commit, can you add Liv as a reviewer as well, once you're ready? Thanks!

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:3]:
```suggestion
- **Managed-service backups**: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for both {{ site.data.products.db }} clusters.
```

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [inline _includes/cockroachcloud/ccloud/backup-types.md:4]:
```suggestion
- {% if page.name == "run-bulk-operations.md" %} **Customer-owned backups**: {% else %} [**Customer-owned backups**](../cockroachcloud/run-bulk-operations.html): {% endif %} You can take manual backups and store them in your [cloud storage buckets](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html) using the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) statement. 
```

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [reply _includes/cockroachcloud/ccloud/backup-types.md:3]: Sorry if this wasn't clear, I intended for the names of the backup types to be in bold rather than the product names here.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [reply _includes/cockroachcloud/ccloud/backup-types.md:4]: Also, double check this in the preview once you have merged this suggestion (or test), but there was a space between "Customer-owned backup" and the colon... I think this was due to the placement of the colon within the if/else block

**Review by kathancox — APPROVED** [review]
LGTM with the small changes suggested here. Liv will need to provide her feedback here before this can be merged.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (livlobo) [inline _includes/cockroachcloud/ccloud/backup-types.md:1]: nit: I think the word 'clsuters' will cause confusion here, recommend removing it and leaving it as "CockroachDB cloud supports two types of ... "

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (livlobo) [inline _includes/cockroachcloud/ccloud/backup-types.md:3]: Recommend slight tweaking:

For serverless and dedicated clusters, Cockroach Labs takes automated backups so that customers' don't have to manage backups themselves. These backups are stored on Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page to learn more about the type and frequency of backups supported on Dedicated and Serverless clusters.

Comment (livlobo) [reply _includes/cockroachcloud/ccloud/backup-types.md:3]: General question - Is the standard across the Docs to represent it as "severless" and "dedicated" (lowercase), as opposed to "Serverless" and "Dedicated" (upper case)?

File: cockroachcloud/backups-page.md
Comment (livlobo) [inline cockroachcloud/backups-page.md:41]: The 'Status' field will be removed pretty soon (over the coming month). Just FYI. 

We may want to file a docs issue to update docs after these two epics are done - https://cockroachlabs.atlassian.net/browse/CRDB-19573 
https://cockroachlabs.atlassian.net/browse/CRDB-19836

File: cockroachcloud/backups-page.md
Comment (livlobo) [inline cockroachcloud/backups-page.md:47]: To view the databases included in the backup **and initiate a restore**, click the number in the [**Databases**](#databases) column

Comment (livlobo) [reply cockroachcloud/backups-page.md:47]: Is there a way to make this page open by default to "Dedicated" tab. Currently, it opens by default on the Serverless tab for me.

File: v22.1/backup-and-restore-overview.md
Comment (livlobo) [inline v22.1/backup-and-restore-overview.md:54]: At the top of this page, there is a breakout of the major sections 

This page provides an overview of the backup and restore features available in CockroachDB:
[Backup and restore types](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-types) and their product availability
[SQL statements](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-sql-statements) for working with backups and restores
[Storage](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-storage) for backups

Should we add "Types of backups supported in CockraochDB Cloud" to this list?

Comment (livlobo) [inline v22.1/backup-and-restore-overview.md:54]: Also, I think we should pull this section of CockroachDB Backup types to the very beginning of this page. So the order would be:

Types of Backups supported by CockroachDB Cloud
[Backup and restore types](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-types) and their product availability
[SQL statements](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-sql-statements) for working with backups and restores
[Storage](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-storage) for backups

**Review by livlobo — COMMENTED** [review]
Added some feedback for this PR in particular, but also overall feedback for this group of pages.

File: _includes/cockroachcloud/ccloud/backup-types.md
Comment (kathancox) [reply _includes/cockroachcloud/ccloud/backup-types.md:3]: Marketing decided that these should be lowercased a month or two back, so docs needed to follow.

File: cockroachcloud/backups-page.md
Comment (ashleykwtam) [reply cockroachcloud/backups-page.md:41]: filed issue here https://cockroachlabs.atlassian.net/browse/DOC-6255

File: v22.1/backup-and-restore-overview.md
Comment (ashleykwtam) [reply v22.1/backup-and-restore-overview.md:54]: changed the order of the page, will work with Kathryn for updating the table and its respective product availability

Comment (kathancox) [reply v22.1/backup-and-restore-overview.md]: Liv's right, this table needs a general overhaul. I've created an issue and will complete this separately as part of the December milestone: https://cockroachlabs.atlassian.net/browse/DOC-6283

File: cockroachcloud/backups-page.md
Comment (kathancox) [reply cockroachcloud/backups-page.md]: @ashleykwtam I think as part of this PR let's leave the serverless filter in the same position, as this matches the rest of the docs in other product areas. The approach to which product should lead in these docs circumstances is a wider product question, beyond this PR. 
Liv, to directly link to the dedicated filter content for a customer, append `?filters=dedicated` to the end of the link.

**Pull request merged**
